teamsupport/models.py

```diff
@@ -1,19 +1,31 @@
 from property_caching import cached_property
 from querylist import QueryList
 
+from teamsupport.services import TeamSupportService
 from teamsupport.errors import MissingArgumentError
 
 
+TICKET_STATUS_NEW = '212203'
+TICKET_TYPE_SUPPORT = '35731'
```

Comment:
I don't see any other way to understand this except ask TeamSupport. Which I don't want to do, because we'll loose one more week waiting for answer.